### PR TITLE
Add/update info for reportValidity() method

### PR DESCRIPTION
**Pull request by jmswisher**
Updated using info from [Caniuse](https://caniuse.com/#feat=constraint-validation).

### CODE REVIEW

File: api/HTMLButtonElement.json [hunk 1]
```diff
@@ -664,6 +664,57 @@
...
       },
+      "reportValidity": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLButtonElement/reportValidity",
+          "support": {
+            "chrome": {
+              "version_added": "40"
+            },
+            "chrome_android": {
+              "version_added": "71"
```
Comment (jpmedley) [inline api/HTMLButtonElement.json]: This should match chrome.

File: api/HTMLButtonElement.json [hunk 2]
```diff
@@ -664,6 +664,57 @@
...
+              "version_added": "10"
+            },
+            "safari_ios": {
+              "version_added": "10"
+            },
+            "samsunginternet_android": {
+              "version_added": "4.0"
+            },
+            "webview_android": {
+              "version_added": "37"
```
Comment (jpmedley) [inline api/HTMLButtonElement.json]: This should also match Chrome.

File: api/HTMLFieldSetElement.json [hunk 3]
```diff
@@ -287,6 +287,57 @@
...
       },
+      "reportValidity": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLButtonElement/reportValidity",
+          "support": {
+            "chrome": {
+              "version_added": "40"
+            },
+            "chrome_android": {
+              "version_added": "71"
```
Comment (jpmedley) [inline api/HTMLFieldSetElement.json]: And here.

File: api/HTMLFieldSetElement.json [hunk 4]
```diff
@@ -287,6 +287,57 @@
...
+              "version_added": "10"
+            },
+            "safari_ios": {
+              "version_added": "10"
+            },
+            "samsunginternet_android": {
+              "version_added": "4.0"
+            },
+            "webview_android": {
+              "version_added": "37"
```
Comment (jpmedley) [inline api/HTMLFieldSetElement.json]: Here.

File: api/HTMLInputElement.json [hunk 5]
```diff
@@ -940,6 +940,57 @@
...
       },
+      "reportValidity": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLButtonElement/reportValidity",
+          "support": {
+            "chrome": {
+              "version_added": "40"
+            },
+            "chrome_android": {
+              "version_added": "71"
```
Comment (jpmedley) [inline api/HTMLInputElement.json]: Match Chrome.

File: api/HTMLInputElement.json [hunk 6]
```diff
@@ -940,6 +940,57 @@
...
+              "version_added": "10"
+            },
+            "safari_ios": {
+              "version_added": "10"
+            },
+            "samsunginternet_android": {
+              "version_added": "4.0"
+            },
+            "webview_android": {
+              "version_added": "37"
```
Comment (jpmedley) [inline api/HTMLInputElement.json]: Match Chrome.

File: api/HTMLOutputElement.json [hunk 7]
```diff
@@ -612,7 +612,7 @@
               "version_added": true
             },
             "chrome_android": {
-              "version_added": true
+              "version_added": "40"
```
Comment (jpmedley) [inline api/HTMLOutputElement.json]: Chrome and webview Android should also be 40.

File: api/HTMLOutputElement.json [hunk 8]
```diff
@@ -630,22 +630,22 @@
...
-              "version_added": false
+              "version_added": "10"
             },
             "samsunginternet_android": {
-              "version_added": true
+              "version_added": "4.0"
             },
             "webview_android": {
-              "version_added": true
+              "version_added": "37"
```
Comment (jpmedley) [inline api/HTMLOutputElement.json]: 40.

File: api/HTMLSelectElement.json [hunk 9]
```diff
@@ -432,6 +432,57 @@
...
       },
+      "reportValidity": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLButtonElement/reportValidity",
+          "support": {
+            "chrome": {
+              "version_added": "40"
+            },
+            "chrome_android": {
+              "version_added": "71"
```
Comment (jpmedley) [inline api/HTMLSelectElement.json]: 40

File: api/HTMLSelectElement.json [hunk 10]
```diff
@@ -432,6 +432,57 @@
...
+              "version_added": "10"
+            },
+            "safari_ios": {
+              "version_added": "10"
+            },
+            "samsunginternet_android": {
+              "version_added": "4.0"
+            },
+            "webview_android": {
+              "version_added": "37"
```
Comment (jpmedley) [inline api/HTMLSelectElement.json]: 40

File: api/HTMLTextAreaElement.json [hunk 11]
```diff
@@ -95,6 +95,57 @@
...
       },
+      "reportValidity": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLButtonElement/reportValidity",
+          "support": {
+            "chrome": {
+              "version_added": "40"
+            },
+            "chrome_android": {
+              "version_added": "71"
```
Comment (jpmedley) [inline api/HTMLTextAreaElement.json]: 40

File: api/HTMLTextAreaElement.json [hunk 12]
```diff
@@ -95,6 +95,57 @@
...
+              "version_added": "10"
+            },
+            "safari_ios": {
+              "version_added": "10"
+            },
+            "samsunginternet_android": {
+              "version_added": "4.0"
+            },
+            "webview_android": {
+              "version_added": "37"
```
Comment (jpmedley) [inline api/HTMLTextAreaElement.json]: 40

File: api/HTMLButtonElement.json (hunk 1 above)
Comment (vinyldarkscratch) [reply api/HTMLButtonElement.json]:
```suggestion
              "version_added": "40"
```

**Pull request merged**